packages/trpc/server/routers/viewer/auth/resendVerifyEmail.handler.ts

```diff
@@ -0,0 +1,19 @@
+import { sendEmailVerification } from "@calcom/features/auth/lib/verifyEmail";
+
+import type { TrpcSessionUser } from "../../../trpc";
+
+type ResendEmailOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+};
+
+export const resendVerifyEmail = async ({ ctx }: ResendEmailOptions) => {
+  const email = await sendEmailVerification({
+    email: ctx.user.email,
+    username: ctx.user?.username ?? undefined,
```

Comment:
```suggestion
    username: ctx.user.username ?? undefined,
```
or maybe make ResendEmailOptions["ctx"]["user"] optional